# Uart Number of bits
# uart_number_of_bits_t - enumeration
uint8 value

#Pre-Defined ENUM Constants
	uint8 uart_bits_Current = 0 #  Use the current number of bits on selected port
	uint8 uart_bits_7       = 1
	uint8 uart_bits_8       = 2
